jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/security/FlowuiVaadinWebSecurity.java

```diff
@@ -34,18 +35,25 @@
 import org.springframework.context.ApplicationContext;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.web.savedrequest.HttpSessionRequestCache;
 import org.springframework.security.web.servlet.util.matcher.MvcRequestMatcher;
-import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+import org.springframework.security.web.util.matcher.*;
 import org.springframework.web.context.WebApplicationContext;
 import org.springframework.web.servlet.handler.HandlerMappingIntrospector;
 
+import java.util.List;
 import java.util.Optional;
 
 /**
  * Provides default Vaadin and Jmix FlowUI security to the project.
  */
 public class FlowuiVaadinWebSecurity extends VaadinWebSecurity {
 
+    /**
+     * Regexp matches all requests that end with ".png" or ".jpe?g".
```

Comment:
What about gif, svg, pdf, etc.?)

Reply:
Added above extensions. But I'm not sure about `.pdf` because this request does not appear in debug.